ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -268,10 +399,52 @@ export default {
         text: {
           en: 'Get In Front',
           de: 'Geh vor den Boss',
-          ko: '정면에 서기',
+          ja: 'アヴァウドの前へ',
+          ko: '보스 앞으로 가세요',
+        },
+      },
+    },
+    {
+      id: 'DelubrumSav Avowed Wrath Of Bozja MT',
```

Comment:
What is MT vs ST?

Also, I think both of these need some sort of conditional.  Maybe only call out if Conditions.caresAboutTankBuster?

Reply:
In Japan DC, only MT(T1)s from each parties share 594E tank cleave, while ST(T2)s for 5BB5.
Change names: MT→1, ST→2